core/service-test-runner/icedfrisby-shields.js

```diff
@@ -19,6 +19,7 @@ const factory = superclass =>
     constructor(message) {
       super(message)
       this.intercepted = false
+      super.networkOn()
```

Comment:
What's the thinking behaind using `networkOn()` rather than `enableNetConnect()` here?

Reply:
IcedFrisby has a `networkOn` method whereas `enableNetConnect` is on nock itself. My understanding (admittedly based on a mental runtime of these functions) was that the factory is dealing with IcedFrisby instances and not nock, and I dunno if IcedFrisby exposes nock's `enableNetConnect`

Reply:
right yep gotcha. Sorted :+1: